packages/terra-status-view/README.md

```diff
@@ -3,7 +3,22 @@
 [![NPM version](https://badgen.net/npm/v/terra-status-view)](https://www.npmjs.org/package/terra-status-view)
 [![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.com/cerner/terra-core)
 
-The status view component provides a customizable icon, message, and buttons in a view.
+## Deprecation Notice
+The terra-status-view npm package is deprecated as of `4.45.0`. terra-status-view will be removed from the repo in a future version, but the package will remain on npm.
+
+### What deprecation means
+Any apps that currently use terra-status-view will continue to work as-is.
+However, the terra-status-view package will no longer continue to be supported / developed.
+
+## Why?
+The reason behind terra-status-view deprecation is to integrate the presentation of overlays with the overall application framework to ensure better accessibility.
+
+## How to replace this package?
+This package can be replaced with [terra-application-status-overlay](https://engineering.cerner.com/terra-ui/application/terra-application/components/application-status-overlay).
+The `terra-application-status-overlay` is constrained to predefined variants to promote unified framework pattern. Hence, it will not support custom variants.
```

Comment:
```suggestion
The `terra-application-status-overlay` is constrained to predefined variants to promote unified framework patterns. Hence, it will not support custom variants.
```